web/beacon-app/cypress/fixtures/user.json

```diff
@@ -1,6 +1,6 @@
 {
-  "email": "",
-  "password": "",
+  "email": "cletus+local@gmail.com",
```

Comment:
Could the data be removed from the json file so that it won't have to be deleted if someone else wants to run a test?